src/components/common/TransactionalFolioNumber.vue

```diff
@@ -1,79 +1,83 @@
 <template>
-  <v-card flat id="transactional-folio-number-container" :class="{ 'invalid-section': !sectionValid }">
+  <v-card flat id="transactional-folio-number-container" :class="{ 'invalid-section': !isValid }">
     <v-row no-gutters>
       <v-col cols="3">
-        <label :class="{ 'error-text': !sectionValid }">
+        <label :class="{ 'error-text': !isValid }">
           <strong>Folio or Reference<br>Number</strong>
         </label>
       </v-col>
       <v-col cols="9">
         <v-text-field
-          filled persistent-hint validate-on-blur hide-details
+          filled persistent-hint
           id="folio-number-input"
           ref="folioNumberInput"
           autocomplete="chrome-off"
           label="Folio or Reference Number (Optional)"
-          v-model="folioNumber"
+          v-model.trim="localFolioNumber"
```

Comment:
"trim" removes leading and trailing whitespace characters when the model variable is updated.

Unfortunately, the component seems to look at the pre-trimmed value when evaluating its rules, so the rules also need to perform trimming. See rules for more details.

Reply:
This was a good find!

Reply:
oh cool.  didn't realize you could add `.trim` to `v-model`